### DISCUSSION TITLE
Should we release 0.70.13?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#79] [Patches]
Released, please refer to #84
---

Current Release: [0.70.12](https://github.com/facebook/react-native/releases/tag/v0.70.12)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. 

#### Local commits to backport to main

1.

**Comment by szymonrybczak** [comment]
Bump CLI to in following packages:
- @react-native-community/cli-hermes@9.3.4
- @react-native-community/cli-platform-android@9.3.4
- @react-native-community/cli@9.3.4

The new version contains fix for broken backward compatibility (related to this: https://github.com/react-native-community/discussions-and-proposals/issues/671).  The issue occurs when the library doesn't have package field in `AndroidMafniest.xml`
